frontend/public/components/utils/details-page.tsx

```diff
@@ -118,11 +118,11 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
               onClick={Kebab.factory.ModifyTolerations(model, resource).callback}
               variant="link"
             >
-              {pluralize(_.size(tolerations), 'Toleration')}
+              {t('details-page~{{count}} toleration', { count: _.size(tolerations) })}
```

Comment:
It looks like this uses inconsistent case with the one just below.

Reply:
This still has inconsistent case with the one below. I assume they should be the same.

Reply:
Whoops, sorry. Should be all set now @spadgett.